localisation/00_zuwc_united_localisation.csv

```diff
@@ -2303,21 +2292,21 @@ text_effect_extra_soul_shard;Has chance to get 
+RULE_HOLY_WARS_RESTRICTED_DESC;Holy War' CB is available only for characters with "Zealous" trait and aggressive religions like the Scarlet Light and Shath'gral.;Les guerres saintes ne sont activ�es que pour pour les personnage ayant le trait 'Z�l�' et une religion belliqueuse comme la Lumi�re �carlate et les Shath'gral.;Der heilige Krieg CB ist nur f�r eifrige Charaktere und aggressive Religionen wie Scarlet Light und Shath'gral verf�gbar.;;;;;;;;;;;x
 RULE_HOLY_WARS_ON;On;R�gle activ�e;Aktiviert;;;;;;;;;;;x
 RULE_HOLY_WARS_ON_DESC;Holy War' CB is available for all.;Les guerres saintes sont activ�es pour tous.;Der heilige Krieg CB ist f�r alle verf�gbar.;;;;;;;;;;;x
 RULE_HOLY_WARS_OFF;Off;R�gle d�sactiv�e;Deaktiviert;;;;;;;;;;;x
 RULE_HOLY_WARS_OFF_DESC;Holy War' CB is disabled.;Les guerres saintes sont d�sactiv�es.;Der heilige Krieg CB ist deaktiviert.;;;;;;;;;;;x
 kolk;Blood of [This.GetFirstNameWithNick];;Blut von [This.GetFirstNameWithNick];;;;;;;;;;;x
-kolk_desc;Kolk was one of the first centaur, directly descended from the union of Therazane and Zaetar. He was the First Khan as well as progenitor of the Kolkar.;;Kolk war einer der ersten Zentauren, der direkt aus der Verbindung von Therazane und Zaetar hervorging. Er war der erste Khan und auch der Stammvater der Kolkar.;;;;;;;;;;;x
+kolk_desc;Kolk was one of the first centaurs, directly descended from the union of Therazane and Zaetar. He was the first Khan as well as progenitor of the Kolkar.;;Kolk war einer der ersten Zentauren, der direkt aus der Verbindung von Therazane und Zaetar hervorging. Er war der erste Khan und auch der Stammvater der Kolkar.;;;;;;;;;;;x
 theradras_worship_opinion;[theradras_worship.GetName] Opinion;;[theradras_worship.GetName] Meinung;;;;;;;;;;;x
 bloodline_effect_shaman_training;Lowered Learning requirement for �GShaman Training�!;;Verringerte Lernvoraussetzung f�r die �GSchamanenausbildung�!;;;;;;;;;;;x
 waylon;Blood of [This.GetFirstNameWithNick];;Blut von [This.GetFirstNameWithNick];;;;;;;;;;;x
-waylon_desc;Waylon was an admiral of Kul Tiras, who led the Stormsong tidesages into the northern valley and drove away the quilboar, as well as established the Shrine of the Storm.;;Waylon war ein Admiral von Kul Tiras, der die Stormsong Gezeitenweisen in das n�rdliche Tal f�hrte und die Stacheleber vertrieb, sowie den Schrein des Sturms errichtete.;;;;;;;;;;;x
+waylon_desc;Waylon was an admiral of Kul Tiras, who led the Stormsong tidesages into the northern valley and drove away the quilboar, as well as established the shrine of the Storm.;;Waylon war ein Admiral von Kul Tiras, der die Stormsong Gezeitenweisen in das n�rdliche Tal f�hrte und die Stacheleber vertrieb, sowie den Schrein des Sturms errichtete.;;;;;;;;;;;x
```

Comment:
**_Shrine_** is a part of the name.
https://wow.gamepedia.com/Shrine_of_the_Storm

Reply:
done